packages/prisma/zod-utils.ts

```diff
@@ -56,3 +56,39 @@ export const stringOrNumber = z.union([
 ]);
 
 export const stringToDayjs = z.string().transform((val) => dayjs(val));
+
+export const bookingCreateBodySchema = z.object({
```

Comment:
Should I look also into re-using those in API instead of generating my own?

Reply:
We can try to merge what we can use from the generated models. But it's good enough for now